frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/hover.dart

```diff
@@ -13,6 +11,10 @@ class FlowyHover extends StatefulWidget {
   final void Function(bool)? onHover;
   final MouseCursor? cursor;
 
+  /// Reset the hover state when the parent widget get rebuild.
+  /// Default to true.
+  final bool? resetHoverOnRebuild;
```

Comment:
If it defaults to true, does this type need to be nullable?

Reply:
You are right, I removed the question mark.